app/src/main/java/org/fossasia/openevent/activities/TracksActivity.java

```diff
@@ -145,6 +148,7 @@ public boolean onQueryTextSubmit(String query) {
     @Override
     public boolean onQueryTextChange(String query) {
         if (!TextUtils.isEmpty(query)) {
+            query = query + "<->" + trackId;
```

Comment:
`query` contains whatever the user types in search field so it's basically the search keyword.
Also, there's a very little chance search keyword would contain `<->` (that's why I choose it in the first place). But still it could happen so I'll strip the keyword of `<->`